### ISSUE TITLE
[DEFECT] Importance of User Experience in Game Design: The Case of Timer Implementation

### ISSUE THREAD

**Issue by EddyChan1129** [COMP3122-2324-Project/Simpsyber#19]
"When I click the "OK" button, it should not immediately start the timer. This is because players need time to strategize and assign tasks to their employees. If the timer starts as soon as the "OK" button is clicked, it could negatively impact the player's experience. They may feel rushed or pressured, which could detract from their enjoyment of the game. It's important to give players the opportunity to plan their strategy effectively before the game begins in earnest."

https://github.com/COMP3122-2324-Project/Simpsyber/assets/156775927/923e9325-65b3-4ede-8c9d-75e9eaa2a6ce

**Comment by louis-hahaha** [comment]
Thats true, I got the same idea, every time start the game, I have to click pause button immediately, it's a bit annoying, 
there might have a way to solve the issue, pause should be set as default in the beginning, and let user finish setting the kanban board, then start the game.

**Comment by ChenManHei** [comment]
> Thats true, I got the same idea, every time start the game, I have to click pause button immediately, it's a bit annoying, there might have a way to solve the issue, pause should be set as default in the beginning, and let user finish setting the kanban board, then start the game.

I think that this idea of setting the game pause as default in the beginning is really good! Players will then do not need to waste their time on clicking the pause button at the beginning and can hence also reduce their time pressure of the game play. However, since the game is pause at the beginning, new players may not be able to know where to start the timer and the game. So, I think a clearer indication of the “play/pause” button can also be added in the beginning of the game to let the new player understand how to start the game. As an example:

![pause_indicate](https://github.com/COMP3122-2324-Project/Simpsyber/assets/158149070/44e438d1-3d15-4501-be18-05be8b667ab0)
